core/lib/l1_contract_interface/src/i_executor/structures/commit_batch_info.rs

```diff
@@ -254,7 +414,6 @@ impl<'a> Tokenizable for CommitBatchInfo<'a> {
                     let blob_commitment = KzgInfo::new(&pubdata).to_blob_commitment();
 
                     let result = std::iter::once(PUBDATA_SOURCE_CALLDATA)
```

Comment:
same here - add a very clear comment that we don't publish pubdata anymore, as this is validium